src/Nancy/StaticContentConfigurationExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Nancy
+{
+    using System.Collections.Generic;
+    using Nancy.Configuration;
+
+    /// <summary>
+    /// Contains <see cref="StaticContentConfiguration"/> configuration extensions for <see cref="INancyEnvironment"/>.  
+    /// </summary>
+    public static class StaticContentConfigurationExtensions
+    {
+        /// <summary>
+        /// Configures <see cref="StaticContentConfiguration"/> 
+        /// </summary>
+        /// <param name="environment"><see cref="INancyEnvironment"/> that should be configured.</param>
```

Comment:
Then we need to fix that, there's a missing `An` at the start of that sentence =)